presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloSplitManager.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.accumulo;
+
+import com.facebook.presto.accumulo.model.AccumuloColumnConstraint;
+import com.facebook.presto.accumulo.model.AccumuloColumnHandle;
+import com.facebook.presto.accumulo.model.AccumuloSplit;
+import com.facebook.presto.accumulo.model.AccumuloTableHandle;
+import com.facebook.presto.accumulo.model.AccumuloTableLayoutHandle;
+import com.facebook.presto.accumulo.model.TabletSplitMetadata;
+import com.facebook.presto.accumulo.model.WrappedRange;
+import com.facebook.presto.spi.ColumnHandle;
+import com.facebook.presto.spi.ConnectorSession;
+import com.facebook.presto.spi.ConnectorSplit;
+import com.facebook.presto.spi.ConnectorSplitSource;
+import com.facebook.presto.spi.ConnectorTableLayoutHandle;
+import com.facebook.presto.spi.FixedSplitSource;
+import com.facebook.presto.spi.connector.ConnectorSplitManager;
+import com.facebook.presto.spi.connector.ConnectorTransactionHandle;
+import com.facebook.presto.spi.predicate.Domain;
+import com.facebook.presto.spi.predicate.TupleDomain;
+import com.facebook.presto.spi.predicate.TupleDomain.ColumnDomain;
+import com.google.common.collect.ImmutableList;
+import io.airlift.log.Logger;
+
+import javax.inject.Inject;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static com.facebook.presto.accumulo.Types.checkType;
+import static java.util.Objects.requireNonNull;
+
+public class AccumuloSplitManager
+        implements ConnectorSplitManager
+{
+    private static final Logger LOG = Logger.get(AccumuloSplitManager.class);
```

Comment:
This variable is unused. Could you run the IntelliJ check for unused variables, and remove them all?